examples/01.register.js

```diff
@@ -20,7 +20,7 @@ const ethPrivKey = envVars.ETH_PRIVATE_KEY
 // NOTE: you can also generate a new key using:`
 // const starkPrivKey = dvf.stark.createPrivateKey()
 const starkPrivKey = envVars.STARK_PRIVATE_KEY
-const infuraURL = `https://ropsten.infura.io/v3/${envVars.INFURA_PROJECT_ID}`
+const infuraURL = `https://goerli.infura.io/v3/${envVars.INFURA_PROJECT_ID}`
```

Comment:
isn't this going to cause issues for staging/prod users of this client since that's still on ropsten. We should ideally parameterize this one, i.e. make the whole link come from envVars instead of just PROJECT_ID

Reply:
+1. I also like the idea of parametrizing the entire provider rather than assuming it is infura